datalad/distribution/install.py

```diff
@@ -310,7 +310,7 @@ def __call__(
                 for p in path:
                     result = Install.__call__(
                         path=p,
-                        source=None,
+                        source=source,
```

Comment:
can't tell really if it is fixing or breaking anything, since there is no test, right? ;)